### ISSUE TITLE
Cannot parse a const type with a modifier

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1041]
For example `int foo(const unsigned int j)` produces:

```
E       DefinitionError: Invalid definition: expected comma between arguments [error at 27]
E         int foo(const unsigned int j)
```

Granted, the const doesn't add any value here, but it is valid C++.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1041
- Originally reported by: Jim Naslund
- Originally created at: 2012-11-17T23:23:44.679

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-11-28 07:00:35+00:00_

fixed on https://bitbucket.org/birkenfeld/sphinx/pull-request/87

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-11-28 07:07:26+00:00_

waiting to accept https://bitbucket.org/birkenfeld/sphinx/pull-request/87

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-11-30 03:46:53+00:00_

fix: cpp domain parser cannot parse a const type with a modifier. closes #1041

→ <<cset 96779ffed80d9a276905ddaa4210d9b190291231>>
